.github/workflows/auto_insp_update.yml

```diff
@@ -39,4 +39,9 @@ jobs:
         delete-branch: true
         commit-message: Auto paper update
         title: "Attempt to auto update papers from INSPIRE"
-        body: "New paper updates found!"
+        body: |
+          New paper updates found!
+
+          Please check the following:
+          - [ ] If this paper used spec, include `used_spec: true` in metadata
```

Comment:
What metadata? Where is it?

Reply:
I can change this to say "the yaml header"

Reply:
Sorry, I still don't understand where it should say `used_spec: true` and what I would need to change. I'm missing some fundamental context on how and where this information is stored, sorry!

Reply:
Check out an example: https://github.com/sxs-collaboration/www_black-holes_org/pull/137/files#diff-f2bb18519d85d6609aee9136c248f63d11b4845ebf862095f22a659632219f04 . This will be in the context of a PR, where you will see a yaml header (in fact the entire thing is just a yaml header).

Reply:
Oh, I see! Maybe say:
```
include ... in the metadata/preamble of the markdown file for the paper
```
? How would I go about doing so? I guess GitHub lets me edit the file?

Reply:
Yeah, you need to push to the branch of the PR

Reply:
I see. It looks like it might be possible for at least some folks to do it only by clicking the triple dots on the file and then "Edit File". Future experiment. Thanks!

Reply:
Previously I have just checked out the PR's branch, edited the md file(s), and then pushed back.